### PR TITLE
create ja_JP Locale

### PR DESCRIPTION
**Pull request by rona-tombo**
it's not perfect, but it can be merged.

### CODE REVIEW

**Comment by pop4959** [comment]
Hello, thank you for making a PR!

First of all, I just want to mention that I changed the target branch for this to dev since master is reserved for release builds (stable versions).

@syokkendesuyo mentioned that he was working on a Japanese locale a while ago. Before merging, he may be interested in reviewing this. I'd say we should give it a day or two and see if he has any feedback for you.

Also, I'm not an expert in encoding, however I don't think you need to use ascii codes for every letter, as most of the other locales do not do this. See for example this [Chinese](https://github.com/pop4959/LWCX/blob/master/src/lang/lwc_cn.properties) locale. Have you tested if the locale works correctly either way?

Lastly, perhaps change the name of the locale code to just "ja" or "jaJP" in order to remain consistent with the naming conventions of the other existing locales. It may otherwise confuse server owners.

**Comment by syokkendesuyo** [comment]
Bukkit has been working with UTF since version 1.9.
Therefore, it is no longer necessary to use ASCII.

Translation comments will be made in separate comments.

File: src/lang/lwc_ja_JP.properties [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+gray_banner=�D�F�̊�
+light_gray_banner=���D�F�̊�
+cyan_banner=�ΐF�̊�
+purple_banner=���F�̊�
+blue_banner=�F�̊�
+brown_banner=���F�̊�
+green_banner=�ΐF�̊�
+red_banner=�ԐF�̊�
+black_banner=���F�̊�
+white_wall_banner=���F�̕ǂɊ|����ꂽ��
```
Comment (syokkendesuyo) [inline src/lang/lwc_ja_JP.properties]: 「壁にかけられた白色の旗」が正しい名称です。
https://minecraft-ja.gamepedia.com/1.13/%E5%B9%B3%E5%9D%A6%E5%8C%96

同様に82-96行目を修正する必要があります。

File: src/lang/lwc_ja_JP.properties [hunk 2]
```diff
@@ -0,0 +1,601 @@
...
+protection.general.notice.protected=%dark_red%���: %white%����%block%��%yellow% %owner%�ɂ���%dark_red%%type%%white%����Ă��܂�.
+
+# Protection has been removed + unregistered
+protection.unregistered=%dark_red%%block%�̕ی�����܂���.
+
+# Enforce access to blocks
+protection.general.locked.password=\
+%dark_red%����%block%�̓p�X���[�h�ی삳��Ă��܂�. \n\
+%gold%%cunlock% <Password>%dark_red% �ƃR�}���h���s���ĉ���Ă�������.
+protection.general.locked.private=%dark_red%����%block%��LGW�ɂ��ĕی삳��Ă��܂�.
```
Comment (syokkendesuyo) [inline src/lang/lwc_ja_JP.properties]: LGWとは何ですか？LWCの間違えだと思われます。

Comment (rona-tombo) [reply src/lang/lwc_ja_JP.properties]: > LGWとは何ですか？LWCの間違えだと思われます。

とりあえずここまでの分は修正しました。

**Comment by syokkendesuyo** [comment]
@rona-tombo 
> Lastly, perhaps change the name of the locale code to just "ja" or "jaJP" in order to remain consistent with the naming conventions of the other existing locales. It may otherwise confuse server owners.

Major issues have been fixed. Finally, you need to change the file name.
Thank you for your contribution.

[**ja**]
大きな問題は修正されました。最後にファイル名の変更をお願いします。


@pop4959 
The problem has been corrected.
Minor issues will be improved in consultation with native speakers.
In that case, create a new pull request.

**Comment by rona-tombo** [comment]
完了しました。
レビューありがとうございました。

@syokkendesuyo Thank you for reviewing!

**Comment by pop4959** [comment]
Thank you for your hard work! Glad to see that syokkendesuyo was able to give you a good review. There is not any expectation that it is perfect right now, as it can be added to and fixed later in additional pull requests. I'm sure Japanese servers will be happy to see this new locale, though. :smiley: 

You can get new build here: <https://ci.ender.zone/job/LWCX/102/>

It will also be included in the next version uploaded to Spigot.

**Pull request merged**